src/commands/code-analyzer/config.ts

```diff
@@ -66,7 +66,7 @@ export default class ConfigCommand extends SfCommand<void> implements Displayabl
 	protected createDependencies(outputFile?: string): ConfigDependencies {
 		const uxDisplay: UxDisplay = new UxDisplay(this, this.spinner);
 		const modelGeneratorFunction = (rawConfig: CodeAnalyzerConfig, ruleSelection: RuleSelection) => {
-			return DummyConfigModel.fromSelection(rawConfig, ruleSelection);
+			return AnnotatedConfigModel.fromSelection(rawConfig, ruleSelection);
```

Comment:
This is the work from the last PR paying off, because all I had to do was swap which ConfigModel implementation is being used and everything just works.